src/galileo/protect.py

```diff
@@ -56,10 +58,16 @@ def invoke(
             metadata=metadata,
             headers=headers,
         )
+        request_dict = request.model_dump()
+        request_dict["prioritized_rulesets"] = request_dict.pop("rulesets", [])
```

Comment:
Note: translation required where `RulesetsMixin` is used since generated model uses `prioritized_rulesets`